checkout-form-manager.js

```diff
@@ -621,7 +628,7 @@ function submitCheckout(e) {
 
     const formData = new FormData(form);
     formData.append('hash', (new URL(document.location)).searchParams.get('hash'));
-    formData.append('offer_id', checkoutElement.getAttribute('offerId'));
+    formData.append('offer_id', checkoutElement.getAttribute('offer_id'));
```

Comment:
This is an oops on my part from before - we should be getting the `offer_id` from the `config`.
```suggestion
    formData.append('offer_id', config['offer_id']);
```